Dockerfile

```diff
@@ -79,8 +79,8 @@ WORKDIR /root/Mobile-Security-Framework-MobSF
 RUN pip3 install -r requirements.txt
 
 #check if Postgres support must be enabled 
-WORKDIR /root/Mobile-Security-Framework-MobSF/
-RUN chmod +x ./postgres_support.sh && ./postgres_support.sh $POSTGRES
+WORKDIR /root/Mobile-Security-Framework-MobSF/scripts
+RUN chmod +x ./postgres_support.sh; sync; ./postgres_support.sh $POSTGRES
```

Comment:
Any particular reason to use sync ?

Reply:
yes if you don't use sync you will have a busy:
```
/bin/sh: 1: ./postgres_support.sh: Text file busy
```

Reply:
ok, thanks for the clarification